src/config/yaml.ts

```diff
@@ -82,6 +82,9 @@ export const USER_ACTIVENESS_MONTHLY_VOLUME_THRESHOLD = toCents(
   yamlConfig.userActivenessMonthlyVolumeThreshold,
 )
 
+export const BRIA_WALLET_NAME = yamlConfig.bria.hotWalletName
+export const BRIA_QUEUES = yamlConfig.bria.queues
```

Comment:
same here, maybe in the main file to consolidate process and yaml configs

Reply:
https://github.com/GaloyMoney/galoy/pull/2662#discussion_r1209421804